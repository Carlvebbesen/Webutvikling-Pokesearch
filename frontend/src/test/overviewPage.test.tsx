import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import ReactSixteenAdapter from 'enzyme-adapter-react-16';
import OverviewPage from '../pages/Overview';
import { MockedProvider } from '@apollo/client/testing';
import { mocks } from './testData';


Enzyme.configure({ adapter: new ReactSixteenAdapter() });

test('renders the App Component', () => {
    const wrapper = shallow(
    <MockedProvider mocks={mocks} addTypename={false}>
    <OverviewPage/>
    </MockedProvider>
    );
   const hp =  wrapper.find("hp")
    expect(hp.prop(name) == "hp");
  });


