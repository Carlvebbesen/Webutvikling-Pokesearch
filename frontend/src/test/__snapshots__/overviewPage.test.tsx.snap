// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the App Component 1`] = `ShallowWrapper {}`;
